app/population/view/daily.njk

```diff
@@ -0,0 +1,65 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+
+{% block pageTitle %}
+  {{ t(pageTitle) }} – {{ super() }}
+{% endblock %}
+
+{% block content %}
+  <header>
+    <h1 class="govuk-heading-xl">
+      {{ resultsAsDailySummary.date | formatDateWithDay }}
+    </h1>
+  </header>
+
+  {%  call appPanel() %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      <h1 class="govuk-heading-m">{{ t("population::daily_view.labels.free_spaces") }}</h1>
+      <h2 class="govuk-heading-xl">{{ t("population::spaces_with_count", {count:resultsAsDailySummary.freeSpaces}) if resultsAsDailySummary.freeSpaces else t("population::daily_view.not_calculated") }}</h2>
+      <p class="govuk-body">{{ t("population::daily_view.free_space_description") }}</p>
+      <p class="govuk-body"><b>{{  t("population::daily_view.labels.last_updated") }} {{  resultsAsDailySummary.updatedAt | formatDateWithDayAndTime if resultsAsDailySummary.updatedAt else t("population::daily_view.not_updated")}}</b></p>
+    </div>
+  </div>
+  {%  endcall %}
+
+  <div class="govuk-grid-row govuk-!-padding-top-4">
+    <div class="govuk-grid-column-one-half">
```

Comment:
This block looks like it contains quite a bit of duplication, especially around markup.

I wonder if we can extract some of this into the controller/a presenter so that view gets a list of section/data groups that it is then responsible for displaying.

From looking at the screenshot I also wonder if we could re-use the [data component](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/master/common/components/data) that we already have to display this information. 

(I'm aware the lack of ability to view all components probably made it harder to realise this existed though)

But it might be worth a chat with @Andy-Millar about re-using that to display this information.

It's actually currently used to render the location here, but can easily be re-used in this instance:

<kbd>![image](https://user-images.githubusercontent.com/3327997/98716488-8bea2d00-2383-11eb-81e0-b489899141f4.png)</kbd>

Another thing to consider would be flipping the number/label order so that the number appears first. This is something that benefits how things read (for example, with a screenreader when there is no visual style). Ie, `5 overnights in` reads more naturally than `overnights in 5`. 

Here's some examples of other parts of govt that use this pattern for data/statistics:

http://alphagov.github.io/digitalmarketplace-frontend-toolkit/statistics.html
https://govuk-elements.herokuapp.com/data/#data-visualisation

And here's another example of GOV.UK Notify summarise counts:

<kbd><img width="1032" alt="dashboard-summary" src="https://user-images.githubusercontent.com/3327997/98716912-3eba8b00-2384-11eb-9abd-17dd67302706.png"></kbd>

Reply:
Not sure I agree with reusing (top screenshot). If we can have the font size-m (not small, as in the card screenshot). If that's not an issue though, then all good to reuse.

Don't flip the label - since we are having the edit state trigger everything into fields but in single column. This page isn't really to be a dashboard - it's just showing ins and outs in a concise way, allowing users to enter / update numbers as required.

Reply:
>Not sure I agree with reusing (top screenshot). If we can have the font size-m (not small, as in the card screenshot). If that's not an issue though, then all good to reuse.

Yes, sorry I meant more just re-use the component in terms of how we write the HTML in the markup. We have a few "modifiers" that we can attach to that component to tweak is visually, for things like font size or font weight so I think we can try and match it visually. But it would be good if we can re-use it from a code perspective.

>Don't flip the label - since we are having the edit state trigger everything into fields but in single column. This page isn't really to be a dashboard - it's just showing ins and outs in a concise way, allowing users to enter / update numbers as required.

Sure. But would it not be okay to still provide a different view when editing? A lot of our other screens don't match exactly to how the data is captured.

Reply:
Reusing code - gotcha all good.

Flipping - no, the user anchors to the data title then the number. Leave as spec-ed please

Reply:
> Flipping - no, the user anchors to the data title then the number. Leave as spec-ed please

@Andy-Millar Ok re unavailable space counts. But...could we maybe flip it for the summary of the total spaces though? Doesn't look like the user enters this figure so doesn't need the form field "anchor" and it feels like using the same pattern used across government would be good in this instance?

So 
```
70
free spaces
```
instead of 
```
Free spaces
70 spaces
```

Reply:
I'd dispute that it is used _across_ government - both patterns are used widely depending on the context.

But...

70
free spaces

...is fine though. 

Would perhaps be better to do away with both options actually, and just go for:

70 free spaces

...all on one line?

Reply:
ie.

<img width="1017" alt="Screenshot 2020-11-12 at 16 11 59" src="https://user-images.githubusercontent.com/22559322/98965341-d7c2e080-2501-11eb-873b-4c0ecf83c012.png">

Reply:
Yep, also an option actually. I just find that it reads better (if you take away the styling) by having the number first instead of second.

Reply:
@teneightfive The `data` component displays its content with different sized text - I think the example you're using is the `card`, which is using some custom styles on `dt/dd` tags - https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/components/card/template.njk#L39-L54

As an example: 

![Screenshot from 2020-11-16 16-12-27](https://user-images.githubusercontent.com/196695/99278229-b1b17f00-2826-11eb-9207-3c3b7e0af748.png)

From left to right we have:

- Data component
- Card component

and then

- Previous implementation

Reply:
@Andy-Millar I've updated the "free spaces panel" to look more like this now:

![Screenshot from 2020-11-16 16-12-48](https://user-images.githubusercontent.com/196695/99278554-1c62ba80-2827-11eb-8b28-1bdc8ed7b1e3.png)

Reply:
Looks good to me